src/Tree.tsx

```diff
@@ -124,8 +124,8 @@ export interface TreeProps<TreeDataType extends BasicDataNode = DataNode> {
   onBlur?: React.FocusEventHandler<HTMLDivElement>;
   onKeyDown?: React.KeyboardEventHandler<HTMLDivElement>;
   onContextMenu?: React.MouseEventHandler<HTMLDivElement>;
-  onClick?: NodeMouseEventHandler;
-  onDoubleClick?: NodeMouseEventHandler;
+  onClick?: NodeMouseEventHandler<TreeDataType>;
+  onDoubleClick?: NodeMouseEventHandler<TreeDataType>;
```

Comment:
There are two generics args, do you only need to declare one explicitly and use the default for the other?

Reply:
I believe so, i've searched for uses of onClick and onDoubleClick and they are only invoked from
- TreeNode:onSelectorClick line 44 and 
- TreeNode:onSelectorDoubleClick line 58

..in both of usages the type is HTMLSpanElement (the default) so I thought it might be best to leave it.